content/knowledge/scalinghistory.md

```diff
@@ -0,0 +1,135 @@
+
+# A Brief History of Decentralised Scaling
+
+This article will cover how the scaling solutions for decentralised systems in the Crypto world have evolved with regards to the consensus algorithms that have been used, what their problems are and how Flare tries to address them.
+We will cover how various blockchains fare in the realm of the [blockchain trilemma](https://vitalik.ca/general/2021/04/07/sharding.html) which roughly states that it is hard to achieve more than 2 ouf 3 from Security, Scalability and Decentralisation.
+
+The concept of Blockchain or Cryptography are not new and have been around before the 2000's.
+Still there was no concept of a truly digital currency/value due to the double spending problem.
+For example, say I have $100 worth of digital value and I send $100 to both Alice and Bob at the same time.
+There used to be no fully robust way to stop this from happening.
+
+## 1. Proof of Work (PoW)
+
+   Then came Satoshi Nakamoto who introduced the concept of Proof of Work (PoW) on top of a Blockchain which acts as a distributed ledger of all the transactions of value/currency.
+   So now imagine I have 100 Bitcoins.
+   There are many machines/nodes which have this information that I have 100 Bitcoins.
+   If I send 50 of them to Alice then certain things happen before it gets "finalised".
+   These things include me sending out this transaction info, some number of machines/nodes accepting it (along with transactions from others), verifying that I am truly the one sending it out and then they solve a complicated problem which takes a lot of computing power.
+   The first one to solve it actually gets to "write" these transactions in the form of a block and that's how it is finalised.
+   A good interactive visualisation of how PoW consensus might work and create a chain of blocks can be found [here](https://youtu.be/_160oMzblY8).
+
+PoW by itself isn’t very new.
```

Comment:
Also please avoid contractions, like `isn't`, `doesn't`, etc. and prefer the full form: is not, does not. Contractions are meant to be informal and do not belong in technical documents IMO

Reply:
@emailtovamos Unless I am mistaken, this one has not been changed.


```suggestion
PoW by itself is not very new.
```